WebApplication/State/OssBucket.cs

```diff
@@ -260,40 +255,10 @@ public async Task<string> TryToCreateSignedUrlForReadAsync(string objectName)
         /// </summary>
         public async Task SmartUploadAsync(string fileName, string objectName, int chunkMbSize = 5)
         {
-            // 2MB is minimal, clamp to it
-            chunkMbSize = Math.Max(2, chunkMbSize);
-            long chunkSize = chunkMbSize * 1024 * 1024;
-
             await using var fileReadStream = File.OpenRead(fileName);
 
-            // determine if we need to upload in chunks or in one piece
-            long sizeToUpload = fileReadStream.Length;
-
-            // use chunks for all files greater than chunk size
-            if (sizeToUpload > chunkSize)
-            {
-                _logger.LogInformation($"Uploading in {chunkMbSize}MB chunks");
-
-                string sessionId = Guid.NewGuid().ToString();
-                long begin = 0;
-                byte[] buffer = new byte[chunkSize];
-
-                while (begin < sizeToUpload-1)
-                {
-                    var bytesRead = await fileReadStream.ReadAsync(buffer, 0, (int)chunkSize);
-
-                    int memoryStreamSize = sizeToUpload - begin < chunkSize ? (int)(sizeToUpload - begin) : (int)chunkSize;
-                    await using var chunkStream = new MemoryStream(buffer, 0, memoryStreamSize);
-                    
-                    var contentRange = $"bytes {begin}-{begin + bytesRead - 1}/{sizeToUpload}";
-                    await UploadChunkAsync(objectName, contentRange, sessionId, chunkStream);
-                    begin += bytesRead;
-                }
-            }
-            else
-            {
-                await UploadObjectAsync(objectName, fileReadStream);
-            }
+            // The underlying APS .NET SDK already does chunking  
+            await UploadObjectAsync(objectName, fileReadStream);
```

Comment:
Is that really same chunking?

Reply:
Yes. 
https://github.com/Autodesk-Forge/forge-api-dotnet-client/blob/3ed6f137cc693414c506bd703e2d2044a0495a0c/src/Autodesk.Forge/Api/ObjectsApi.cs#L4832

Reply:
Also, the API that was used underneath the old chunking code is now deprecated and will be gone: https://aps.autodesk.com/en/docs/data/v2/reference/http/buckets-:bucketKey-objects-:objectKey-resumable-PUT/